Kickstarter-iOS/Views/Controllers/PaymentMethodsViewController.swift

```diff
@@ -117,24 +123,49 @@ internal final class PaymentMethodsViewController: UIViewController, MessageBann
 
     self.present(nav, animated: true, completion: nil)
   }
-}
 
-extension PaymentMethodsViewController: UITableViewDelegate {
+  // MARK: - Private Helpers
 
-  func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
-    return 0.1
+  private func configureHeaderFooterViews() {
+    if let header = SettingsTableViewHeader.fromNib(nib: Nib.SettingsTableViewHeader) {
+      header.configure(with: Strings.Any_payment_methods_you_saved_to_Kickstarter())
+
+      let headerContainer = UIView(frame: .zero)
+      headerContainer.addSubview(header)
+
+      self.tableView.tableHeaderView = headerContainer
+
+      header.constrainEdges(to: headerContainer)
+
+      _ = header.widthAnchor.constraint(equalTo: self.tableView.widthAnchor)
+        |> \.priority .~ .defaultHigh
+        |> \.isActive .~ true
+    }
+
+    if let footer = PaymentMethodsFooterView.fromNib(nib: Nib.PaymentMethodsFooterView) {
+      footer.delegate = self
+
+      let footerContainer = UIView(frame: .zero)
+      footerContainer.addSubview(footer)
+
+      self.tableView.tableFooterView = footerContainer
+
+      footer.constrainEdges(to: footerContainer)
+
+      _ = footer.widthAnchor.constraint(equalTo: self.tableView.widthAnchor)
```

Comment:
@justinswart updating the priority here, in the header's constraints and in the xibs fixed all the auto-layout warnings (there were also some happening when moving into landscape mode).

Priorities updated:
- height constraint on the `PaymentFooterView xib` -> `.defaultHigh`
- width constraint on footer and header -> `.defaultHigh`
- top and bottom constraints on the `UILabel` in `SettingsTableViewHeader` -> `.defaultHigh`

Tested on iPhone X (12.1), iPhone 6 (11.4), iPhone 5s (10.3.1)